docs/src/components/layouts/utility-page-layout/utility-class-tables/text-demonstration.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropType from 'prop-types';
+import UtilityClassTitleBar from '../utility-class-example-title-bar';
```

Comment:
Kill this. The utility-info-table serves this purpose and is already drawn above the classes list.